parser/parsetypes/parsetypes.go

```diff
@@ -1,14 +1,16 @@
 package parsetypes
 
 import (
-	"github.com/activecm/rita/config"
 	"strings"
+
+	"github.com/activecm/rita/config"
 )
 
 //BroData holds a line of a bro log
 type BroData interface {
 	TargetCollection(*config.StructureTableCfg) string
 	Indices() []string
+	ConvertFromJSON()
```

Comment:
Sorry if I'm being dense, but does this gets called for everything, regardless if the file was json or a bro log? Could we just split this into a different type? Maybe at least a clarifying comment on it would help for future development, I understand that it might be too difficult to split up.

Reply:
It's defined on the interface that is used for both tsv and json logs. But it's only called when a json log is imported, not a tsv one (https://github.com/activecm/rita/blob/146-json-support/parser/fileparser.go#L221).

I can certainly add a comment. Would `// ConvertFromJSON is called when importing from JSON logs` work, or did you have something in mind?

Reply:
yup, that would be great, and could we add some comments to the parseline function if statements just to visually separate and clarify the two choices (json and bro)?